_layouts/main.html

```diff
@@ -0,0 +1,265 @@
+        }
+
+        .bd-sidebar {
+            grid-area: sidebar;
+        }
+
+        .bd-toc {
+            grid-area: toc;
+        }
+
+        .bd-toc nav ul {
+            padding-left: 0;
+            margin-bottom: 0;
+            list-style: none;
+        }
+
+        .bd-toc nav li {
+            margin-bottom: 0.25rem;
+        }
+
+        .bd-toc nav a {
+            display: block;
+            padding: 0.125rem 0 0.125rem 0.75rem;
+            color: inherit;
+            text-decoration: none;
+            border-left: 0.125rem solid transparent;
+        }
+
+        .bd-toc nav a:not(:hover) {
+            text-decoration: none;
+        }
+
+        .bd-toc nav a.active {
+            font-weight: 700;
+        }
+
+        .bd-toc nav a:hover,
+        .bd-toc nav a.active {
+            color: #000;
+            border-left-color: #ff7900;
+        }
+
+        .bd-links-link {
+            padding: 0.1875rem 0.5rem;
+            margin-top: 0.125rem;
+            margin-left: 1.375rem;
+            color: rgba(0, 0, 0, 0.65);
+            text-decoration: none;
+        }
+
+        .bd-links-link.active {
+            font-weight: 600;
+        }
+
+        @media (min-width: 768px) {
+            .bd-main {
+                display: grid;
+                grid-template-areas:
+                    "intro"
+                    "toc"
+                    "content";
+                grid-template-rows: auto auto 1fr;
+                gap: inherit;
+            }
+
+            .bd-toc-collapse {
+                display: block !important;
+            }
+        }
+
+        @media (min-width: 1024px) {
+            .bd-layout {
+                display: grid;
+                grid-template-areas: "sidebar main";
+                grid-template-columns: 1fr 5fr;
+                gap: 1.25rem;
+            }
+
+            .bd-main {
+                grid-template-areas:
+                    "intro toc"
+                    "content toc";
+                grid-template-rows: auto 1fr;
+                grid-template-columns: 4fr 1fr;
+            }
+
+            .bd-toc {
+                position: -webkit-sticky;
+                position: sticky;
+                top: 7rem;
+                right: 0;
+                z-index: 2;
+                height: calc(100vh - 7rem);
+                overflow-y: auto;
+            }
+        }
+    </style>
+</head>
+
+<body>
+
+    <header class="sticky-top">
+        <nav class="navbar navbar-dark bg-dark navbar-expand-lg" aria-label="Global navigation">
+            <div class="container-xxl">
+                <div class="navbar-brand me-auto me-lg-4">
+                    <a class="stretched-link" href="#">
+                        <img src="https://c.woopic.com/logo-orange.png" width="50" height="50"
+                            alt="ODS Android - Back to Home" loading="lazy">
+                    </a>
+                    <h1 class="title">ODS Android</h1>

```

Comment:
There are 2 occurrences of "Android" that shouldn't be in this repository

Reply:
You are right and we know it. We had some trouble figuring ot the best way to organize and separate things and that's why I asked you for a meeting about the architecture of the project 😉